website/static/css/custom.css

```diff
@@ -49,4 +49,5 @@ div.terminal pre {
   background-color: black;
   color: white;
   padding: 20px;
+  overflow: auto
```

Comment:
That's for the ScalaCheck integration page: http://weaver-site.surge.sh/weaver-test/docs/scalacheck 

The error message is hella long and breaks rendering.